root.tf

```diff
@@ -94,10 +62,77 @@ module "frontend" {
 module "keycloak" {
   app_name                    = "keycloak"
   source                      = "./modules/keycloak"
+  alb_dns_name                = module.keycloak_alb.alb_dns_name
+  alb_target_group_arn        = module.keycloak_alb.alb_target_group_arn
+  alb_zone_id                 = module.keycloak_alb.alb_zone_id
+  dns_zone_id                 = local.dns_zone_id
+  dns_zone_name_trimmed       = local.dns_zone_name_trimmed
   environment                 = local.environment
   environment_full_name       = local.environment_full_name_map[local.environment]
   common_tags                 = local.common_tags
   database_availability_zones = local.database_availability_zones
   az_count                    = 2
   region                      = local.region
 }
+
+module "alb_logs_s3" {
+  source        = "./tdr-terraform-modules/s3"
+  project       = "tdr"
```

Comment:
Should this be `var.project` like the others?

Reply:
Thanks, corrected